<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFire</class>
 <widget class="QDialog" name="DialogFire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px solid #595959;
background-color: #FCA6A6;
border-radius: 16px;
</string>
  </property>
  <widget class="QPushButton" name="BTN_FORCED_STOP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;함초롬돋움&quot;;
background-color: #999999;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>강제종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_POINT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{ 
   border: 0px solid
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="PB_PROGRESS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     text-align: center;
 }

 QProgressBar::chunk {
     background-color: #FFFFFF;
     border-radius: 5px;
 }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_MENU">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  background-color:lightyellow;
  font: 25pt &quot;함초롬돋움&quot;; 
  font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LB_BASKET_NAME">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{ 
   border: 0px solid;
   font: 15pt &quot;함초롬돋움&quot;;
   background-color:white;
  border-radius: 20px;
} 
</string>
   </property>
   <property name="text">
    <string>#1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
